docs/open-source/index.mdx

```diff
@@ -1,8 +1,18 @@
 ---
 title: Open source
-description: TODO - filler text so that the SEO check doesn't complain!
+description: Qiskit open-source introduction
 ---
 
 # Open source
 
-Blah blah
+Since 2016, IBM Quantum&trade; has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
+This groundbreaking initiative includes open-sourcing key parts of the quantum software stack and introducing Qiskit, a versatile quantum computing framework.
+Today, Qiskit is the most popular quantum framework, thanks in part to its vibrant community.
+The Qiskit community is a dynamic and welcoming group of developers, scientists, and enthusiasts collaborating to advance software, research, education, and business through quantum computing.
+By making this transformative technology open and accessible, we accelerate innovation and discover new applications.
+
+As quantum technology evolves, contributions from the open-source [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) projects and developers are crucial in shaping its future.
+
+By joining the Qiskit community, you can participate in the development of quantum technologies.
+Whether you are a seasoned quantum programmer or beginning your quantum journey, Qiskit provides the support and inspiration needed to push boundaries.
+We welcome all kinds of contributions, from code in pull requests to community support on [Slack](https://docs.quantum.ibm.com/support#qiskit), ensuring everyone can play a role in advancing quantum technology.
```

Comment:
@Eric-Arellano can/should we use an internal link here?

```suggestion
We welcome all kinds of contributions, from code in pull requests to community support on [Slack](/support#qiskit), ensuring everyone can play a role in advancing quantum technology.
```

Reply:
Yes, thanks.